brave/browser-laptop
# Clean up formatting for bitcoin wallet backup

Fix #4807

Auditors @bsclifton @bbondy 

Test Plan:

1. Open Brave
2. Go to Preferences > Payments
3. Click Advanced Settings
4. Try both Printing and Saving your backup keys. They should look like the screenshots below.

<img width="1280" alt="screen shot 2016-11-27 at 3 17 00 pm" src="https://cloud.githubusercontent.com/assets/490294/20652781/17daa638-b4b5-11e6-9182-6b6311cc29ae.png">
<img width="650" alt="screen shot 2016-11-27 at 3 17 19 pm" src="https://cloud.githubusercontent.com/assets/490294/20652782/17dceede-b4b5-11e6-95fb-caec9aed479b.png">
